packages/turf-clusters/README.md

```diff
@@ -0,0 +1,172 @@
+# @turf/clusters
+
+# getCluster
+
+Get Cluster
+
+**Parameters**
+
+-   `geojson` **[FeatureCollection](http://geojson.org/geojson-spec.html#feature-collection-objects)** GeoJSON Features
+-   `filter` **Any** Filter used on GeoJSON properties to get Cluster
+
+**Examples**
+
+```javascript
+var geojson = turf.featureCollection([
+    turf.point([0, 0]),
+    turf.point([2, 4]),
+    turf.point([3, 6]),
+    turf.point([5, 1]),
+    turf.point([4, 2])
+]);
+
+// Create a cluster using K-Means (adds `cluster` to GeoJSON properties)
+var clustered = turf.clustersKmeans(geojson);
+
+// Retrieve first cluster (0)
+var cluster = turf.getCluster(clustered, {cluster: 0});
+```
+
+Returns **[FeatureCollection](http://geojson.org/geojson-spec.html#feature-collection-objects)** Single Cluster filtered by GeoJSON Properties
+
+# clusterEachCallback
+
+Callback for clusterEach
+
+**Parameters**
+
+-   `cluster` **[FeatureCollection](http://geojson.org/geojson-spec.html#feature-collection-objects)** The current cluster being processed.
+-   `clusterValue` **Any** Value used to create cluster being processed.
+-   `currentIndex` **[number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)** The index of the current element being processed in the array.Starts at index 0
+-   `geojson`  
+-   `property`  
+-   `callback`  
+
+Returns **void** 
+
+# clusterEach
+
+clusterEach
+
+**Parameters**
+
+-   `geojson` **[FeatureCollection](http://geojson.org/geojson-spec.html#feature-collection-objects)** GeoJSON Features
+-   `property` **([string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) \| [number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number))** GeoJSON property key/value used to create clusters
+-   `callback` **[Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)** a method that takes (cluster, clusterValue, currentIndex)
+
+**Examples**
+
+```javascript
+var geojson = turf.featureCollection([
+    turf.point([0, 0]),
+    turf.point([2, 4]),
+    turf.point([3, 6]),
+    turf.point([5, 1]),
+    turf.point([4, 2])
+]);
+
+// Create a cluster using K-Means (adds `cluster` to GeoJSON properties)
+var clustered = turf.clustersKmeans(geojson);
```

Comment:
Although their primary use is actual clustered points, I mean as output of a clustering module, these functions could be useful to identify any group of points with a common property, even among points where not all have said property (like identify all points with a certain `marker-symbol`).
I think it could be beneficial to the user to see examples that are not necessarily related to clustered (in the above mentioned meaning) points, in order to suggest other uses and applications for this module's functions.